clipboard.go

```diff
@@ -0,0 +1,20 @@
+package fyne
+
+// Clipboard represents a single button in a MouseEvent
+type Clipboard struct {
+	Window
```

Comment:
I wonder if the public Clipboard and public Window field should be hidden behind a public interface?
The need for a window is a glfw implementation detail I think so we shouldn't expose it...

Reply:
Agreed, added in 949f61fe7d10d029b8bd8068202d1f8c17144999